runelite-client/src/main/java/net/runelite/client/ui/overlay/worldmap/WorldMapOverlay.java

```diff
@@ -57,6 +57,7 @@
 	private static final int TOOLTIP_PADDING_HEIGHT = 1;
 	private static final int TOOLTIP_PADDING_WIDTH = 2;
 	private static final int TOOLTIP_TEXT_OFFSET_HEIGHT = -2;
+	private static final int ICON_EDGE_SNAPPED_INSET = -15;
```

Comment:
Where does this value come from? This won't look right for all icons; we should calculate this for each icon's size

Reply:
I arbitrarily chose it from the current plugins that use it (Clues and Party) as a decent value to use. I thought about calculating it for each icon separately, and in hindsight it would probably be better to do it like that.